easybuild/easyconfigs/p/PyTables/PyTables-3.5.2_fix-LIBS.patch

```diff
@@ -0,0 +1,14 @@
+PyTables expects that LIBS is a list of library names, and will add '-l' by itself.
```

Comment:
This patch is identical to `PyTables-3.5.2-fix-libs.patch`, which is already in the tree. Please switch to it and remove the duplicate from your PR.